tests/core/test_data.py

```diff
@@ -184,3 +184,10 @@ def test_is_not_nlu_file_with_json():
     io.write_text_file('{"test": "a"}', file)
 
     assert not data.is_nlu_file(file)
+
+
+def test_get_source_file_name():
+    assert data.get_source_file_name("") == ""
+    assert data.get_source_file_name("/tmp/stories.md") == "stories.md"
+    assert data.get_source_file_name("/tmp/123_stories.md") == "stories.md"
+    assert data.get_source_file_name("/tmp/123_my_stories.md") == "my_stories.md"
```

Comment:
The original stories are copied to a temporary directory and the filename get prefixed with an unique ID. This was the reason to strip file name so the original story file name was printed.

I agree to leave the file name intact so there is still a reference to the story (in the temporary directory) which failed e.g.
```
## happy path > /tmp/tmp73u056dx/4f8a5df888ec4e96bafbf9dad54c624d_stories2.md
```
Removed the function which stripped the file name and made changes to pass the file name instead of stripped one.